### ISSUE TITLE
Composer dependencies

### ISSUE THREAD

**Issue by marchampson** [spatie/laravel-ray#9]
This is very likely to be an issue my end, running PHP 7.4.11 and Laravel 7.22.4, when running compser require I get:

```
Using version ^1.0 for spatie/laravel-ray
./composer.json has been updated
Running composer update spatie/laravel-ray
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - spatie/laravel-ray 1.0.0 requires spatie/ray dev-master -> satisfiable by spatie/ray[dev-master].
    - spatie/laravel-ray[1.0.1, ..., 1.0.2] require spatie/ray ^1.0 -> satisfiable by spatie/ray[1.0.0, 1.0.1].
    - spatie/ray[dev-master, 1.0.0, ..., 1.0.1] require symfony/console ^4.2|^5.2 -> found symfony/console[v4.2.0-BETA1, ..., 4.4.x-dev, v5.2.0-BETA1, ..., 5.x-dev] but the package is fixed to v5.1.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires spatie/laravel-ray ^1.0 -> satisfiable by spatie/laravel-ray[1.0.0, 1.0.1, 1.0.2].
```

If I run with -W for all dependencies I get even more errors. I've had successful composer updates today so don't want to go changing too many things.

Thanks

**Comment by marchampson** [comment]
I've just tried installing on a brand new project and it works, so this is definitely something in my existing project. Strange.

**Comment by axelitus** [comment]
Check your dependencies, here's the error:

```
spatie/ray[dev-master, 1.0.0, ..., 1.0.1] require symfony/console ^4.2|^5.2 -> found symfony/console[v4.2.0-BETA1, ..., 4.4.x-dev, v5.2.0-BETA1, ..., 5.x-dev] but the package is fixed to v5.1.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
```

Ray requires Symfony Console 5.2 onwards, but in your dependencies the version of this library is "fixed" to 5.1 (from your composer.lock file).

Two things I see you could do after checking your composer.json allows for Symfony Console 5.2 onwards:
1. Run `composer update` and see if nothing breaks and the requirement gets updated.
2. Remove your composer.lock file and try again

Hope this helps

**Comment by marchampson** [comment]
Been a long day! I thought I had run update, ran it and all ok now. Thanks @axelitus for taking a look.
